test/core/fixtures/transformation/validation.undeclared-variable-check/undeclared-shorthand-property/exec.js

```diff
@@ -0,0 +1 @@
+({foo})
```

Comment:
Nope, it's just what the ESTree AST spec says to do. Populating `value` with `undefined` or `null` isn't really an option and you can't fake an AST node.